### PR TITLE
Update TROUBLESHOOTING.md

### PR DESCRIPTION
**Pull request by myzeprog**
Resolving #492

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=494) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=494) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-php?pullRequest=494) it.</sub>

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -99,3 +100,14 @@ If you're getting the following error while using this library:
 It is most likely due to a linebreak in your API key. Passing your key through `trim` should fix this:
 
 `$apiKey = trim($apiKey)`
+
+<a name="request-body"></a>
+## Viewing the Request Body
+
+When debugging or testing, it may be useful to exampine the raw request body to compare against the [documented format](https://sendgrid.com/docs/API_Reference/api_v3.html).
```
Comment (Braunson) [inline TROUBLESHOOTING.md]: Spelling mistake, `exampine` should be `examine` 👍

Comment (myzeprog) [reply TROUBLESHOOTING.md]: Done

**Comment by thinkingserious** [comment]
Hello @myzeprog,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**